### DISCUSSION TITLE
Integrations with other tools such as SmartLead/Clay etc

### DISCUSSION THREAD

**Discussion by DuffCorp** [twentyhq/twenty#7206] [Ideas]
Integrations with these tools would be a key deciding factor at my business as it is just as important to us not only the ability of how we can use the CRM for managing the contacts but how we can use them within campaigns etc and then having them talk to each other much like smart lead and HubSpot does is proving a key feature for us to have that communication between them. Being able to see automatically replies and keep track of a conversation is proving key.

**Comment by BOHEUS** [comment]
@DuffCorp thanks for the suggestion, soon it'll be possible to write an integration once SDK is stable and v1.0 is released (in the meantime, feel free to check out docs https://docs.twenty.com/developers/extend/apps/getting-started), I'm closing this discussion in favor of collective discussion of integrations https://github.com/twentyhq/twenty/discussions/16735
